src/orquestra/sdk/_base/cli/_ui/_presenters.py

```diff
@@ -273,6 +273,13 @@ def print_configs_list(
         """
         Print a list of stored configs.
         """
+        if not configs:
+            click.echo(
+                "No remote configs available. Create new config using "
+                "`orq login -s <remote uri>` command"
+            )
+            return
```

Comment:
**issue**: this is an error scenario, not a happy path. We have another way of handling errors, because there are more considerations to keep in mind, e.g. the CLI should exit with a non-zero code. Please consider using the `LoginAction`'s `exception_presenter`.

Reply:
Another reason to refactor this: `if not configs` is error handling code. Presenter is the "view" layer. Error handling shouldn't be done in the view layer.

Reply:
I disagree. Empty list is still a list. Why printing empty list should be considered error scenario

Reply:
My bad. I misread the description. I thought this was about a better error message, but it's just a different way of presenting an empty list.